docs/SelfDiagnostics.MD

```diff
@@ -0,0 +1,65 @@
+# [Advanced] How to enable self diagnostics for ApplicationInsights.Kubernetes
+
+## Enable self-diagnostics
+
+We understand ApplicationInsights.Kubernetes is far from perfect. Enabling self-diagnostics will help find [issues](https://github.com/microsoft/ApplicationInsights-Kubernetes/issues) easier. Please feel free to open issues and we will be glad to look into them.
```

Comment:
The first sentence suggests our confidence in this package is very low.

Consider saying something along the lines of:
1. Enabling self-diagnostics will help troubleshoot issues and determine whether the problem is with service code/configuration, or with the `ApplicationInsights.Kubernetes` package itself.
1. If problem with AI.K package is suspected, please open an issue on GitHub and we will look into it.

Reply:
Good suggestions!